common/src/main/java/com/redhat/cloud/notifications/models/dto/v1/endpoint/EndpointTypeDTO.java

```diff
@@ -0,0 +1,24 @@
+package com.redhat.cloud.notifications.models.dto.v1.endpoint;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum EndpointTypeDTO {
```

Comment:
EndpointType still contains `@Schema` (OpenAPI) and `@JsonProperty` annotations. Shouldn't they be moved here?

Reply:
Yes, true! Moved all the annotations here